youtube_dl/extractor/sverigesradio.py

```diff
@@ -0,0 +1,69 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..compat import compat_str
+from ..utils import (
+    try_get,
+    int_or_none,
+)
+
+
+class SverigesRadioBaseIE(InfoExtractor):
+    _GEO_COUNTRIES = ['SE']
```

Comment:
Add a test for geoblocked bypassable audio.

Reply:
Oh i misunderstood the meaning of this, assumed it was informational in some way. It don't knof of any bypass so i'm removing it